package.json

```diff
@@ -99,6 +98,7 @@
     "has-hover": "^1.0.1",
     "has-passive-events": "^1.0.0",
     "is-mobile": "^4.0.0",
+    "maplibre-gl": "^4.3.2",
```

Comment:
It might be an idea to start with one of the initial versions of `maplibre-gl` which is more similar to `mapbox-gl` v1 and after it rendered properly, then we upgrade the version to introduce new features.

Reply:
Considering the baseline changes, I was wondering if we use an older version of `mablibre` here, the diffs would possibly be less?